src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -301,39 +298,39 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
             {
               displayText: 'Low to High',
               onClick: () => setSortBy(SortBy.LowToHigh),
-              icon: <NonRarityIcon width="28" height="28" color={vars.color.blue400} />,
+              icon: <NonRarityIcon width="20" height="20" viewBox="2 2 22 22" color={vars.color.blue400} />,
               reverseIndex: 2,
             },
             {
               displayText: 'High to Low',
               onClick: () => setSortBy(SortBy.HighToLow),
-              icon: <NonRarityIcon width="28" height="28" color={vars.color.blue400} />,
+              icon: <NonRarityIcon width="20" height="20" viewBox="2 2 22 22" color={vars.color.blue400} />,
               reverseIndex: 1,
             },
             {
               displayText: 'Rare to Common',
               onClick: () => setSortBy(SortBy.RareToCommon),
-              icon: <RarityIcon width="28" height="28" color={vars.color.blue400} />,
+              icon: <RarityIcon width="20" height="20" viewBox="2 2 24 24" color={vars.color.blue400} />,
               reverseIndex: 4,
             },
             {
               displayText: 'Common to Rare',
               onClick: () => setSortBy(SortBy.CommonToRare),
-              icon: <RarityIcon width="28" height="28" color={vars.color.blue400} />,
+              icon: <RarityIcon width="20" height="20" viewBox="2 2 24 24" color={vars.color.blue400} />,
               reverseIndex: 3,
             },
           ]
         : [
             {
               displayText: 'Low to High',
               onClick: () => setSortBy(SortBy.LowToHigh),
-              icon: <NonRarityIcon width="28" height="28" color={vars.color.blue400} />,
+              icon: <NonRarityIcon width="20" height="20" viewBox="2 2 22 22" color={vars.color.blue400} />,
```

Comment:
if these are re-used can you make a single instance at the top and then use the variable?

Reply:
good idea done.